lib/nwi/network_information.h

```diff
@@ -0,0 +1,286 @@
+/*
```

Comment:
Why is there an Apple header included? Is this not part of the macOS SDK?

Reply:
It's not; the header is distributed in a separate tarball. The symbols are, however, part of the SDK libraries.

Reply:
I'm not enthusiastic about having to ship an Apple header for macOS support, but the licence is permissive, so I guess it's fine.